src/models/otp.ts

```diff
@@ -3,14 +3,20 @@ import mongoose from "mongoose";
 // create an interface representing document in mongoDB
 interface IOtp {
   recipient: string;
+  cs: string;
```

Comment:
what is cs ? please use an informatif name

Reply:
cs is the sender of the OTP validation, is it better to be called the sender?